src/ReactPixiFiber.js

```diff
@@ -119,7 +119,7 @@ export function cancelTimeout(id) {
 }
 
 export function clearContainer(container) {
-  container.removeChildren();
+  container && container.removeChildren();
```

Comment:
We use more verbose syntax in this repository:
```suggestion
  if (container) {
    container.removeChildren();
  }
```